reviews/models/mixins.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from transitions import Machine
+
+from django.db import models
+
+from reviews import workflow
+
+
+class ReviewProviderMixin(models.Model):
+    """A reviewed/moderated collection of objects.
+    """
+
+    review_workflow = models.CharField(max_length=15, choices=workflow.WORKFLOW_CHOICES, default=workflow.PRE_MODERATION)
+    review_comments_public = models.BooleanField(default=False)
+    review_comments_anonymous = models.BooleanField(default=True)
+
+    class Meta:
+        abstract = True
+
+
+class ReviewableMixin(models.Model):
+    """Something that may be included in a reviewed collection and is subject to a reviews workflow
+    """
+
+    state = models.CharField(max_length=15, db_index=True, choices=workflow.STATE_CHOICES, default=workflow.PENDING)
```

Comment:
Could we just wrap that model in an object that just forwards changes to `.state` to `.review[s]_state`?
Actually, it might be cleaner to just have a separate object that receives that hooks and handles state changes with a property?